index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Telnet</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <h2>“Hello World, we are software developers!” </h2>
+    <header id="header" class="container">
+        <div class="logo">
+            <img id="header-img" src ="https://upload.wikimedia.org/wikipedia/commons/b/b6/PuTTY_icon_128px.png">
+        </div>
+        
+
+     <nav id="nav-bar">
+        
+            <ul>
+                <a class="nav-link" href="#Plan">Plans</a>
+                <a class="nav-link" href="#Services">Services</a>
+                <a class="nav-link" href="#Premiun">Premiun Content</a>
+                <a class="nav-link" href="#Support">Support</a>
+            </ul>
+     </nav>
+
+
+    </header>
+    <section id="Plan"><center><h1>Plans</h1></center>
+        
+        
+
+        <div class="row">
+            <div class="column">
+                <center><h2>100mb</h2></center>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta ratione repellat corrupti nesciunt. Tenetur placeat, alias distinctio debitis at, labore quaerat, quam fugiat aperiam dolorum quis voluptate enim voluptatum consequatur!</p>
+            </div>
+            <div class="column">
+                <center><h2>150mb</h2></center>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta ratione repellat corrupti nesciunt. Tenetur placeat, alias distinctio debitis at, labore quaerat, quam fugiat aperiam dolorum quis voluptate enim voluptatum consequatur!</p>
+            </div>
+            <div class="column">
+                <center><h2>300mb</h2></center>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta ratione repellat corrupti nesciunt. Tenetur placeat, alias distinctio debitis at, labore quaerat, quam fugiat aperiam dolorum quis voluptate enim voluptatum consequatur!</p>
+            </div>
+          </div>
+        
+        
+
+        
+
+        
+    </section>
+    <br>
+    <br>
+    <br>
+    <br>
+    <section id="Services"><center><h1>SERVICES</h1></center>
+
+        <div class="row">
+            <div class="column">
+                <center><h2>INTERNET</h2></center>
```

Comment:
Please make sure to use justify-content instead of using a center tag and not to leave too much spaces between code